src/__tests__/media-query-test.js

```diff
@@ -210,15 +211,27 @@ describe('Media query tests', () => {
       }
     }
 
+    @Radium({matchMedia})
+    class TestComponent extends Component {
```

Comment:
Sure, this could certainly be abstracted for tests. My preference, though, is for test to be as self-contained as possible.